File: TRABALHO/src/dados/Filme.java
package dados;

public class Filme extends Conteudo {
    private int duracao; //minutos

    public Filme(String t, String gen, String desc, int a, int dur) {
        super(t, gen, desc, a);
        this.duracao = dur;
    }


    public Filme(){}
    public int getDuracao() {
        return duracao;
    }

    public void setDuracao(int duracao) {
        this.duracao = duracao;
    }

    @Override
    public String toString() {
        return String.format("• Título: %s \n• Descrição: %s \n• Gênero: %s \n• Ano: %d \n• Duração: %d minutos \n\n‣ Elenco Principal: %s \n\n‣" +
                " Elenco Secundário: %s", titulo, descricao, genero, ano, duracao, this.getElenco1(), this.getElenco2());
    }
}

File: Lista2/Lista2Ex5/dados/Filme.py
class Filme:
    def __init__(self):
        self.titulo = ""
        self.ano = 0
        self.clasInd = ""
        self.nota = 0.0

    def __str__(self):
        return "Titulo: {} - Ano: {} - Classificação indicativa: {} - Nota: {}".format(self.titulo,self.ano,self.clasInd,self.nota)

